config/settings/base.py

```diff
@@ -393,6 +393,7 @@ def env(name, default=NoDefaultValue, type_=str):
 ACCOUNT_UNIQUE_EMAIL = False
 ACCOUNT_EMAIL_VERIFICATION = "none"
 SOCIALACCOUNT_ADAPTER = "metecho.oauth2.adapter.CustomSocialAccountAdapter"
+SOCIALACCOUNT_LOGIN_ON_GET = True
```

Comment:
@jerivas The newer version of django-allauth [adds this setting](https://django-allauth.readthedocs.io/en/latest/release-notes.html#section-1), defaulting to `False`. It seems like that would be good to support, but I'm not sure the best way to do that. If I try to just `POST` via XHR, I get a CORS error:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://github.com/login/oauth/authorize?client_id=<...>&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Faccounts%2Fgithub%2Flogin%2Fcallback%2F&scope=user%3Aemail+read%3Auser+public_repo&response_type=code&state=<...>. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 404.
```

Reply:
I think you're meant to POST when hitting the allauth endpoint on our own site. I added a form-based implementation that works on my end on 59c94d1506d13defcda88389ae4702551b9ef5fb

Reply:
So it looks like that broke frontend tests because they expect direct manipulation of `window.location` when clicking the button. Not sure how to test plain form submissions in Jest 😅

Reply:
Can the test add an onsubmit handler to the form or something like that?

Reply:
> I think you're meant to POST when hitting the allauth endpoint on our own site. I added a form-based implementation that works on my end on [59c94d1](https://github.com/oddbird/Metecho/commit/59c94d1506d13defcda88389ae4702551b9ef5fb)

Yep, makes sense -- I figured a <form> might be the way to go. (I was POSTing to the right place, but trying via XHR instead of form submission.)